VTUL/iawa
# Move mirador.js out of application to a static location

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1796) (:star:)

# What does this Pull Request do? (:star:)
This PR moves static JavaScript and CSS files out of IAWA to a static location so to avoid the overhead when bringing up the application.

# What's the changes? (:star:)

* Point JavaScript and CSS to a static link
* Removed JavaScript and CSS from public folder

# How should this be tested?
* Bring up the application in production mode
* Run task ./bin/rake iawa:batch_import RAILS_ENV=production
* See Mirador Viewer properly displays images for each item 

# Additional Notes:

* What branch to be used for testing this PR? LIBTD-1796

# Interested parties
Tag (@yinlinchen ) interested parties

(:star:) Required fields
